src/main/scala/dsptools/numbers/algebra_types/helpers/Sign.scala

```diff
@@ -69,6 +69,7 @@ object Sign {
     override def sign(a: Sign): Sign = a
     def signum(a: Sign): ComparisonBundle = ComparisonHelper(a.zero, a.neg)
     def abs(a: Sign): Sign = if (a == Negative) Positive else a
+    def context_abs(a: Sign): Sign = if (a == Negative) Positive else a
```

Comment:
If I don't have context_abs here then I get a SignAlgebra must be abstract error.